### ISSUE TITLE
(med) overall issue - how much stick to important for accessibility vs additional benefits

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-easy-checks#213]
Pages: several
Location: "Who depends on"
Related issue: (high) Who depends on -> Why it's important #208

Do we want to keep this to the succinct primary accessibility needs? Or include all relevant accessibility needs?

Do we want to include the additional benefits that are not particular accessibility needs? And if so, do we want to clearly distinguish? Do we want that in a more info section at the bottom?

Note that in the "In Brief" sections in the Understanding docs, we chose to keep to the succinct primary accessibility issue, and let the rest of the page provide more info. For some topics, we have a good pointer to the info elsewhere -- e.g., in the Tutorials or media resource. For some topics, maybe not.

Example: [Who depends on Captions?](https://deploy-preview-131--wai-easychecks.netlify.app/test-evaluate/easy-checks/captions/#who-depends-on-captions ) has:
> [accessibility, primary] People who are Deaf, hard-of-hearing, or otherwise have difficulty hearing, or cannot hear, the sound depend on captions to provide audio content.

> [accessibility, less _(although I'm one of them:)_] Some people who process written information better than audio information also use captions.

> [borderline] Some people also like to use captions to be sure they’re not missing the dialogue due to a speaker’s accent or quiet or fast speech, or loud background sounds.

> [additional benefit] People in noisy environments or who have to keep noise levels down can also benefit.

> [additional benefit] Captions also benefit people watching videos in their non-native language and for people learning a new language.

For captions, we have a good resource that lists those **and even more benefits**: [Used by People With and Without Disabilities](https://www.w3.org/WAI/media/av/users-orgs/#situations) and [Benefits to Organizations](https://www.w3.org/WAI/media/av/users-orgs/#benefits)

<hr>

For cases such as captions where we have a good place to point to, I'm leaning towards sticking to the easy to understand accessibility needs, and pointing elsewhere for the additional benefits. Yet, making that clear and maybe in the Why it's important section. e.g.,

> _(To learn how captions also benefit people without disabilities and benefit organizations that provide captions, see [User Experiences and Benefits to Organizations in Making Audio and Video Media Accessible](https://www.w3.org/WAI/media/av/users-orgs/).)_

For topics that don't have a good pointer, I lean towards including the additional benefits in a More Info section (possibly combined with Learn More?

**Comment by shawna-slh** [comment]
related issue https://github.com/w3c/wai-easy-checks/issues/185
